src/fr-window.c

```diff
@@ -4160,6 +4160,25 @@ file_list_drag_begin (GtkWidget          *widget,
 	return TRUE;
 }
 
+static void
+grab_widget (GtkWidget *widget)
+{
+
+    GdkWindow *window;
+    GdkDisplay *display;
+    GdkSeat *seat;
+
+    g_return_if_fail (widget != NULL);
+    window = gtk_widget_get_window (widget);
+    display = gdk_window_get_display (window);
+
+    seat = gdk_display_get_default_seat (display);
+    gdk_seat_grab (seat, window,
+                   GDK_SEAT_CAPABILITY_ALL, TRUE,
```

Comment:
That's probably sufficient
```suggestion
                   GDK_SEAT_CAPABILITY_ALL_POINTING, TRUE,
```